src/Symfony/Bundle/FrameworkBundle/Resources/config/assets.xml

```diff
@@ -7,16 +7,17 @@
     <services>
 
         <service id="assets.packages" class="Symfony\Component\Asset\Packages">
-            <argument type="service" id="assets.empty_package" /> <!-- default package -->
+            <argument type="service" /> <!-- default package -->
             <argument type="collection" /> <!-- named packages -->
         </service>
 
-        <service id="assets.empty_package" class="Symfony\Component\Asset\Package" public="false">
```

Comment:
removing this service is a BC break, as people might reference it in their config. Please keep it